engine/src/flutter/shell/platform/windows/public/flutter_windows.h

```diff
@@ -245,6 +249,10 @@ FLUTTER_EXPORT HWND FlutterDesktopViewGetHWND(FlutterDesktopViewRef view);
 FLUTTER_EXPORT IDXGIAdapter* FlutterDesktopViewGetGraphicsAdapter(
     FlutterDesktopViewRef view);
 
+// Returns the ID3D11Device used for rendering or nullptr in case of error.
```

Comment:
We should call out the lifetime of the returned value. Should we be pessimistic and state the lifetime is only as long as the view? (This isn't strictly true today, the lifetime is as long as the engine. But being conservative here seems useful?)
 
```suggestion
// Returns the ID3D11Device used for rendering or nullptr in case of error.
//
// The ID3D11Device's lifetime is the same as the |view|'s.
```

Reply:
hmm since multi-window is going to be a thing ig would be right to say `engine` here